tests/sles4sap/wizard_hana_install.pm

```diff
@@ -109,20 +109,18 @@ sub run {
             die "Failed";
         }
     }
+
+    # Enable autostart of HANA HDB, otherwise DB will be down after the next reboot
+    select_console 'root-console' unless check_var('DESKTOP', 'textmode');
+    my $hostname = script_output 'hostname';
+    file_content_replace("/hana/shared/${sid}/profile/${sid}_HDB${instid}_${hostname}", '^Autostart[[:blank:]]*=.*' => 'Autostart = 1');
```

Comment:
Same  comment as in `tests/sles4sap/hana_install`: don't know if this setting would impact the HanaSR tests.

Reply:
HanaSR tests doesn't install HANA with the wizard, so no impact. But I could easily add a `HA_CLUSTER` condition, will do.